src/internal/m365/onedrive/url_cache_test.go

```diff
@@ -89,10 +90,37 @@ func (suite *URLCacheIntegrationSuite) TestURLCacheBasic() {
 
 	nfid := ptr.Val(newFolder.GetId())
 
+	collectorFunc := func(
+		ctx context.Context,
```

Comment:
To condense code: since none of these values are used, you can replace their names with `_`, or even take out the var names entirely.
```suggestion
		context.Context,
```